modules/programs/rcm.nix

```diff
@@ -0,0 +1,36 @@
+{ config, pkgs, user, lib, ... }:
+
+with lib;
+with lib.strings;
+let
+  cfg = config.programs.rcm;
+
+  stringifyList = l:
+    if isList l then ''"${concatStringsSep " " l}"'' else stringifyList [ l ];
+
+  toConfigFile = generators.toKeyValue {
+    mkKeyValue = k: v: "${toUpper k}=${stringifyList v}";
+    listsAsDuplicateKeys = false;
+  };
+in {
+  options = {
+    programs.rcm = {
+      enable = mkEnableOption "rcm";
+      settings = mkOption {
+        type = with types; attrsOf (either str (listOf str));
+        default = { };
+        description = lib.mdDoc ''
+          Submodule defining the .rcrc file. Mapping of the key value pairs for rcms config.
+        '';
+      };
+    };
+
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = with pkgs; [ rcm ];
+    home.file.".rcrc" = { text = toConfigFile cfg.settings; };
```

Comment:
Arguably could be set to an arbitrary location with the `RCRC`  environment variable (which is done in other modules, e.g: GnuPG).

Reply:
Ha, never used that but thats a great idea. Thanks for the hint.